code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -646,14 +647,22 @@
 			return
 		..()
 
-//Armor: Gives the target a basic cultist combat loadout
+/obj/item/melee/blood_magic/construction/proc/check_menu(mob/user)
+	if(!istype(user))
+		return FALSE
+	if(user.incapacitated() || !user.Adjacent(src))
+		return FALSE
+	return TRUE
+
+
```

Comment:
What's up with this change? This actually is from tgstation/tgstation#50620.

Reply:
Cherry picked on accident